index.html

```diff
@@ -14,138 +14,201 @@
-     </section>
-     <footer>
-       <h2 id="pm">LET'S CONNECT</h2>
-       <div class="social-media-icons">
-         <a href="https://x.com/@chukwus618"><img src="icons/x.svg" alt="x-handle"></a> 
-         <a href="https://linkedin.com/in/smart-chukwu-89b7072b0/"><img src="icons/Linkedin.svg" alt="Linkedin-handle"></a>
-         <a href="https://medium.com/@chukwus618"><img src="icons/median.svg" alt="Median-handle"></a>
-         <a href="https://github.com/chukwus618"><img src="icons/github.svg" alt="github-handle"></a>
-         <a href="https://wellfound.com/u/smart-chukwu"><img src="icons/wellfound.svg" alt="Wellfound-handle"></a>
-       </div>
-      </footer>
-   </section>
+   </nav>
+    <section class="information">
+      <div class="information-container">
+        <section class="text">
+          <h1>I'm Promise.</br>Glad to see you!</h1>
+          <p class="primary-text">I'm a software developer! I can help you build a product, feature or website look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p> 
+        </section>
+        <footer>
+          <h2 id="pm">LET'S CONNECT</h2>
+          <div class="social-media-icons">
+          <a href="https://x.com/@chukwus618"><img src="icons/x.svg" alt="x-handle"></a> 
+          <a href="https://linkedin.com/in/smart-chukwu-89b7072b0/"><img src="icons/Linkedin.svg" alt="Linkedin-handle"></a>
+          <a href="https://medium.com/@chukwus618"><img src="icons/median.svg" alt="Median-handle"></a>
+          <a href="https://github.com/chukwus618"><img src="icons/github.svg" alt="github-handle"></a>
+          <a href="https://wellfound.com/u/smart-chukwu"><img src="icons/wellfound.svg" alt="Wellfound-handle"></a>
+        </footer>
+      </div>
+    </section>
  </article>
  <article class="work-section">
-   <section class="works">
+    <section id="work1" class="works">
+     <img class="snapshoot" src="icons/snapshoot5.png" alt="A snapshot">
+     <div class="project-card-details">
+       <h3>Tonic</h3>
+       <ul>
+         <li class="caption1">CANOPY</li>
+         <div id="circle"></div>
+         <li><span>Back End Dev</span></li>
+         <div id="circle"></div>
+         <li><span>2015</span></li>
+       </ul>
+       <p class="text3">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+       <ul>
+         <div class="tag1">
+           <li class="tag">HTML</li>
+         </div>
+         <div class="tag2">
+           <li class="tag">CSS</li>
+         </div>
+         <div class="tag3">
+           <li class="tag">JavaScript</li>
+         </div>
+       </ul>
+       <button type="button">See project</button>
+     </div>
+    </section>
```

Comment:
- [ ] Lastly, this is almost not necessary when you've adjusted the width by increasing the percentage between 768 - 1279px, this will give the content more breathing space, and visitors would be able to read or interact with it as they please without stress.

![image](https://github.com/user-attachments/assets/53dd0ce9-4f6b-4bb2-be8e-921ce39a797a)